apps/evm-bridge/src/components/deposit/DepositForm.tsx

```diff
@@ -173,7 +178,13 @@ export function DepositForm({
                         type={ButtonType.Primary}
                         icon={<SwapAccount className="rotate-90 -scale-x-100" />}
                         onClick={() => {
-                            setValue(BridgeFormInputName.IsFromLayer1, !isFromLayer1);
+                            const newIsFromLayer1 = !isFromLayer1;
```

Comment:
can we rename the const `newIsFromLayer1` to `updatedDirection` and maybe extract this code from template?